spec/migrations/20221201184037_change_resources_to_be_float_in_resource_pools_spec.rb

```diff
@@ -0,0 +1,43 @@
+require_migration
```

Comment:
Since the migration is using Rails built-in methods and nothing custom we don't need a spec at all, so this can be deleted.